DataProcessing/datax-host/pom.xml

```diff
@@ -51,7 +51,7 @@ SOFTWARE
 
     <groupId>com.microsoft.datax</groupId>
     <artifactId>datax-host_2.4_2.11</artifactId>
-    <version>1.2.8-SNAPSHOT</version>
+    <version>1.2.9-SNAPSHOT</version>
```

Comment:
Also change the datax dependency versions to 1.2.9-SNAPSHOT on line 121 and 126. Basically for datax-core_2.4_2.11 and datax-utility_2.4_2.11 dependencies

Reply:
updated that